light-client/tests/support/single_step/sequential/commit/_more_than_two_third_vals_sign.json

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Same question about gitgnore.

Reply:
Same response. Gitignore only activates with an underscore in the folder name. Here, there is an underscore in the file name.

Reply:
Ah OK. I see. Thanks for the clarification.